tests/acceptance/features/bootstrap/OCSContext.php

```diff
@@ -940,6 +940,24 @@ public function getOCSResponseStatusCode(ResponseInterface $response):string {
 		);
 	}
 
+	/**
+	 * Parses the xml answer to return data items from ocs response
```

Comment:
```suggestion
```

I think the function name says it all.

Reply:
I put the comment just in case people don't  confuse data items for the whole XML DATA